src/allotropy/allotrope/schema_mappers/adm/plate_reader/benchling/_2023/_09/plate_reader.py

```diff
@@ -394,7 +393,7 @@ def _get_fluorescence_measurement_document(
                 device_control_document=[
                     FluorescencePointDetectionDeviceControlDocumentItem(
                         device_type=metadata.device_type,
-                        detection_type=ReadMode.FLUORESCENCE.value,
+                        detection_type=measurement.detection_type,
```

Comment:
mostly a curiosity question, it is surprising to me that a fluorescence measurement document isn't always of type `FLUORESCENCE` (and likewise for the other modes) for example, if a `FluorescencePointDetectionDeviceControlDocumentItem` is actually of type `LUMINESCENCE` or something, isn't it going to be missing some fields that it needs that are in `LuminescencePointDetectionDeviceControlDocumentItem`?

Reply:
I had the same thought - I originally thought that in this PR I was going to get rid of `MeasurementType` and use `detection type` to determine the type of measurement document to create.

Looking into the existing implementation though, sometimes detection_type is more specific (which is when it really makes sense to even have this field). For example, in the `Revvity Kaleido` parser, we read the detection type from the input data experiment type.